src/calendar/header/style.js

```diff
@@ -26,7 +26,9 @@ export default function(theme={}) {
       ...appStyle.arrowStyle
     },
     arrowImage: {
-      tintColor: appStyle.arrowColor
+      tintColor: appStyle.arrowColor,
+      width: 16,
+      height: 27
```

Comment:
I don't like setting width and height as hard-coded values, but I think we can allow passing them in `appStyle`. Meaning they will be undefined unless passed: `width: appStyle.arrowWidth, height: appStyle.arrowHeight`.

Reply:
Works for me!